students/mimalvarez_pintor/lesson02/Grid_Printer.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Apr 13 21:42:59 2020
+
+@author: miriam
+"""
+# Printing a grid part 1
+n= 1
```

Comment:
Missing space before assignment operator: n = 1, here and elsewhere in this code